CHANGELOG.md

```diff
@@ -1,6 +1,204 @@
-# Changelogs
```

Comment:
The changelog should not be included in these changes. We'll generate a changelog in the next release

Reply:
Thanks for explaination